tests/test_manual.py

```diff
@@ -0,0 +1,76 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the National Aeronautics and Space Administration.  All Rights Reserved.
+
```

Comment:
Could you add a description to the top here in three quotes. Describe the purpose of this file, how it's not run automatically, and why